indico/modules/designer/client/js/index.js

```diff
@@ -240,11 +244,28 @@ import {$T} from 'indico/utils/i18n';
     var itemStyles = _.filter([item.bold ? 'bold' : null, item.italic ? 'italic' : null]);
 
     // Change the selectors so that they match the properties of the item
+    $('#text-color-selector').val(item.color);
+    var textColorPreview = $('#text-color-preview');
```

Comment:
Maybe we could use `const`/`let` in new code, even if the rest of the file is very old and still uses `var`?